packages/test/test-gc-sweep-tests/src/dataObjectWithChildDataObject.ts

```diff
@@ -26,39 +23,40 @@ export class RootDataObjectWithChildDataObject extends DataObjectWithCounter {
     protected async hasInitialized(): Promise<void> {
         await super.hasInitialized();
         await this.createAndReferenceChild();
-        // TODO: Start logic goes here after we agree on what it should be.
+        this.start();
     }
 
-    // TODO: Make private - left public so the build passes
-    public async unreferenceChild() {
+    private unreferenceChild() {
         assert(this.child !== undefined, `Child should be defined when unreferencing!`);
-        this.unreferencedChild = this.child;
         this.child.stop();
         this.root.delete(this.childKey);
-        this.unrefTimeStampMs = Date.now();
     }
 
     private async createAndReferenceChild() {
         this.child = await dataObjectWithCounterFactory.createInstance(this.context.containerRuntime);
-        await this.referenceChild(this.child.handle);
+        this.root.set(this.childKey, this.child.handle);
+        this.child.start();
     }
 
-    private async referenceChild(handle: IFluidHandle<DataObjectWithCounter>) {
-        this.unreferencedChild = this.child;
-        this.root.set(this.childKey, handle);
-    }
+    protected async sendOps() {
```

Comment:
This function should be called `startActivity` or `start` since its doing more than just sending ops.
Also, add comments that describes what this function is supposed to since its not apparent.

Reply:
I don't think this should be called `start`...  Maybe it could be called `run` or something.  I guess it does make sense to choose something more generic though.

Reply:
Choosing run